app/assets/javascripts/services/ems_cloud/arbitration_profile_data.js

```diff
@@ -0,0 +1,11 @@
+ManageIQ.angular.app.service('arbitrationProfileDataFactory', ['API', function(API) {
+  var urlBase = '/api/arbitration_profiles';
+
+  this.getArbitrationProfileData = function (ap_id) {
+    if(angular.isDefined(ap_id)) {
+      return API.get(urlBase + '/' + miqUncompressedId(ap_id))
+    }
+  };
+  return this;
```

Comment:
Uh, this `return this;` looks a bit misleading .. not needed anyway (it was back when this was a factory, not a service)